k8s/nginx-ingress/stage-platform-ingress-resource.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
Thanks @danielogen, This had to be done after upgrading the Kubernetes cluster cc @123MwanjeMike

Reply:
Yes @NoahNsimbe.
This was done so all is well